spec/intl.html

```diff
@@ -502,7 +504,7 @@ <h1>PartitionDateTimePattern ( _dateTimeFormat_, _x_ )</h1>
       <emu-alg>
         1. <ins>Let _x_ be ? HandleDateTimeValue(_dateTimeFormat_, _x_).</ins>
```

Comment:
I think you forgot to add `[[timeZone]]: *undefined*` to the Record returned by HandleDateTimeOthers.

Reply:
Oops!